tf2onnx/tfonnx.py

```diff
@@ -1800,6 +1833,8 @@ def mknode(op_type, inputs, opname, **kwargs):
     "Square": (square_op, []),
     "SquaredDifference": (squareddifference_op, []),
     "Softmax": (direct_op, ["Softmax"]),
+    "LogSoftmax": (direct_op, ["LogSoftmax"]),
```

Comment:
try to find their place in alphabetic order?

Reply:
resolved